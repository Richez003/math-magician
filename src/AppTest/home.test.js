import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../page/home'

it('renders correctly', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  
