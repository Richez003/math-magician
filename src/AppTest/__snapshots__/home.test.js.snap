// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="home-wrapper"
>
  <h2>
    Welcome To Our Page
  </h2>
  <div
    className="body-wrapper"
  >
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita!!
    </p>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita!!
    </p>
  </div>
</section>
`;
